7.5/enable-azure-service-bus.ps1

```diff
@@ -0,0 +1,164 @@
+param(
+    [Parameter(Mandatory = $true)]
+    [string] $connectionString
+)
```

Comment:
can't guarantee I understood everything. But from the looks of it content is fine!

Reply:
Thanks.